api/v1/hostprofile_types.go

```diff
@@ -435,7 +435,7 @@ type CommonInterfaceInfo struct {
 
 	// PTPRole defines the ptp role as master, slave, or none
 	// +kubebuilder:validation:Enum=master;slave;none
-	PTPRole *string `json:"ptpRole,omitempty"`
```

Comment:
I would not prefer to request the user to update their existing deployment-config.yaml for a none value caused by adding mandatory CRD, though it is the simplest approach to the developers.
Can we check&fix it during composing the default& final profiles?

Reply:
No updates requried here.
`MergeProfiles(defaults, profile)` already handles it.